benchmarks/polys.py

```diff
@@ -341,4 +373,28 @@ class TimeGCD_QuadraticNonMonicGCD(_TimeGCD):
 
 class TimeGCD_SparseNonMonicQuadratic(_TimeGCD):
     GCDExampleCLS = _SparseNonMonicQuadratic
-    params = [(1, 3, 5), ('expr', 'dense', 'sparse')]
\ No newline at end of file
+    params = [(1, 3, 5), ('expr', 'dense', 'sparse')]
+
+
+class _TimeGaussianInt(_TimeOP):
+    """Benchmarks for GCDs method when input is Gaussian Integer"""
```

Comment:
This class just duplicates `_TimeGCD` so it is unnecessary.

Reply:
Removed it.